### PR TITLE
Stabilize Chat Live Tests

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

File: sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadAsyncClientTest.java [hunk 1]
```diff
@@ -408,6 +409,9 @@ public void canSendTypingNotificationWithResponse(HttpClient httpClient) {
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    @DisabledIfEnvironmentVariable(
+        named = "SKIP_CHAT_READ_RECEIPT_LIVE_TESTS",
```
Comment (angiurgiu) [inline sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadAsyncClientTest.java:413]: nit: should we make this more generic, if we expand this to other tests as well in the future?

**Review by angiurgiu — APPROVED** [review]

**Review by jbeauregardb — APPROVED** [review]

File: sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadAsyncClientTest.java (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatThreadAsyncClientTest.java]: Done :) thanks

**Comment by minnieliu** [comment]
\azp run java - communication - ci

**Comment by minnieliu** [comment]
\azp run java - communication - ci

**Comment by minnieliu** [comment]
\azp run java - communication - ci

**Pull request merged**